go/pkg/cas/upload.go

```diff
@@ -22,6 +39,9 @@ type UploadInput struct {
 	// TODO(nodir): add PreserveSymlinks.
 }
 
+// FilePredicate is a condition for a file/dir.
```

Comment:
This is not used in this PR now.

Reply:
removed all predicate-related stuff from this CL